nullaway/build.gradle

```diff
@@ -26,11 +27,20 @@ configurations.maybeCreate("epJavac")
 sourceCompatibility = "1.8"
 targetCompatibility = "1.8"
 
+configurations.all {
+    resolutionStrategy {
+        // Force needed, since:
+        // a) There are newer versions of CF than the one we forked
+        // b) It seems x.y.z-name < x.y.z for gradle dependency resolution purposes.
+        force deps.build.checkerDataflow
+    }
+}
+
 dependencies {
     compileOnly deps.apt.autoValue
     compileOnly deps.apt.autoService
 
-    compile deps.build.errorProneCheckApi
+    shadow deps.build.errorProneCheckApi
```

Comment:
For some reason, saying `shadow` here means "exclude this and its dependencies from the shadowJar"... yes, really...

Reply:
weird...